recipes/glib/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Needed to get appropriate response to g_get_system_data_dirs():
+export CFLAGS="-DCONDA_SYSTEM_DATA_DIRS=\\\"${PREFIX}/share\\\""
```

Comment:
I see now. This is being used with `system-data-dirs.patch`. Maybe we should reference that patch here. Sorry for being confused.